raiden/network/transport/matrix/transport.py

```diff
@@ -658,6 +666,7 @@ def _handle_invite(self, room_id: _RoomID, state: dict):
             else:
                 break
         else:
+            assert last_ex is not None
```

Comment:
This can never fail. 
Unless you wan't to keep it in as a defensive, 'refactor proof' reminder.

Reply:
Again, `mypy` workaround, it can't detect the last_ex under `else` from `for` is always set

Reply:
Blahrg, that seems to be a theme with static checkers. PyCharm also has problems with transitive logic.